src/test/kotlin/Step1.kt

```diff
@@ -0,0 +1,20 @@
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.types.shouldBeTypeOf
+
+class Step1 : StringSpec({
+    "named arguments" {
+        val p = Person("name", 20, "nickname")
```

Comment:
named arguments가 의도하는 바는 
```suggestion
        val p = Person(name= "name", age =  20, nickname= "nickname")
```

위와 같이 파라미터를 이름기반 매핑이 가능한지를 보는거에요. 추가적으로 생성자파라미터 순서대로가 아니여도 정상적으로 생성이 되는지도 고려될 수 있겠죠.
그리고 테스트 대상인 p도 약어를쓰기보다는 제대로 된 변수명을 지어보면 좋을 것 같아요.